mongo/mongo_test.go

```diff
@@ -617,38 +639,40 @@ func (s *MongoSuite) TestInstallMongodServiceExists(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 
 	c.Check(s.data.Installed(), gc.HasLen, 0)
-	s.data.CheckCallNames(c, "Installed", "Exists", "Running")
+	s.data.CheckCallNames(c, "Running")
```

Comment:
Why has existing behaviour changed? We need to ensure that non-snap service setup is exactly how it has always been